package.json

```diff
@@ -1,102 +1,21 @@
-    "watch": "gulp watch",
-    "heroku-postbuild": "npm run build && npm run build:bower",
-    "start": "DEBUG=ferreiro:* nodemon --exec babel-node ./bin/www"
-  },
-  "dependencies": {
-    "aws-sdk": "^2.125.0",
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.0",
-    "babel-loader": "^7.1.2",
-    "babel-preset-env": "^1.6.0",
-    "babel-preset-es2015": "^6.24.1",
-    "bcrypt": "^1.0.3",
-    "bluebird": "^3.5.0",
-    "body-parser": "^1.18.2",
-    "bower": "^1.8.2",
-    "compression": "^1.7.1",
-    "connect-flash": "^0.1.1",
-    "cookie-parser": "^1.4.3",
-    "debug": "^3.1.0",
-    "del": "^3.0.0",
-    "email-templates": "^3.0.3",
-    "express": "^4.16.1",
-    "express-recaptcha": "^3.0.0",
-    "express-session": "^1.15.6",
-    "force-ssl-heroku": "^1.0.2",
-    "gulp": "^3.9.1",
-    "gulp-autoprefixer": "^4.0.0",
-    "gulp-babel": "^7.0.0",
-    "gulp-concat": "^2.6.1",
-    "gulp-cssnano": "^2.1.2",
-    "gulp-file-cache": "0.0.1",
-    "gulp-imagemin": "^3.4.0",
-    "gulp-minify": "^1.0.0",
-    "gulp-minify-css": "^1.2.4",
-    "gulp-rename": "^1.2.2",
-    "gulp-sass": "^3.1.0",
-    "gulp-sass-glob": "^1.0.8",
-    "gulp-uglify": "^3.0.0",
-    "helmet": "^3.8.2",
-    "html-toc": "^2.0.0",
-    "imagemin": "^5.3.1",
-    "install": "^0.10.1",
-    "lodash": "^4.17.11",
-    "mailchimp-api-v3": "^1.7.1",
-    "marked": "^0.3.6",
-    "mongoose": "^4.11.13",
-    "mongoose-paginate": "^5.0.3",
-    "mongoose-permalink": "^2.0.0",
-    "mongoose-simple-random": "^0.4.1",
-    "morgan": "^1.9.0",
-    "multer": "^1.3.0",
-    "multer-s3": "^2.7.0",
-    "nodemailer": "^2.3.0",
-    "nodemon": "^1.12.1",
-    "npm": "^6.4.1",
-    "passport": "^0.4.0",
-    "passport-local": "^1.0.0",
-    "pug": "^2.0.0-beta6",
-    "react": "^16.5.2",
-    "react-dom": "^16.5.2",
-    "replaceall": "^0.1.6",
-    "request": "^2.83.0",
-    "sanitize-html": "^1.14.1",
-    "serve-favicon": "^2.4.5",
-    "standard": "^10.0.3",
-    "turbolinks": "^5.0.3",
-    "validator": "^9.0.0",
-    "webpack": "^4.19.1"
-  },
-  "devDependencies": {
-    "npm-check": "^5.4.5"
-  }
+    "private": true,
+    "name": "ferreiro.me",
+    "engines": {
+        "node": "8.10.0"
+    },
+    "scripts": {
+        "client": "cd packages/ferreiro-client && PORT=5000 yarn start",
+        "server": "cd packages/ferreiro-server && yarn start",
+        "devClient": "cd packages/ferreiro-client && PORT=5000 yarn dev",
+        "devServer": "cd packages/ferreiro-server && yarn dev ",
+        "buildAssets": "cd packages/ferreiro-server && yarn build",
+        "dev": "yarn buildAssets && concurrently --kill-others-on-fail \"yarn devServer\" \"yarn devClient\"",
+        "start": "yarn install && yarn buildAssets && concurrently --kill-others-on-fail \"yarn server\"  \"yarn client\"",
+        "heroku-postbuild": "npm run build && npm run build:bower"
+    },
+    "workspaces": [
+        "packages/*"
+    ],
+    "devDependencies": {}
```

Comment:
Good point! I added concurrently to devDependencies in @ferreiro/server. Forgot to add it here.

Side question: Last time I checked, heroku doesn't care about devDependencies and it only gets the normal dependencies. So I think I should add concurrently here as well.

Side question 2: If concurrently doesn't appear as a dependency in the root package, is it because is getting the dependency that `ferreiro-server/package.json` instanciates? I don't think I have a local copy.

Reply:
Yes, part of `yarn workspaces` is that it pull deps from the inner packages to the top level. Then we it has conflict it put specific version in the `node_modules` relative to that workspace.

Reply:
cool. Is there a way to run a script that checks that all the required dependencies for each of the project packages are installed? Otherwise, it looks we can have cases where yarn workspaces installs the dependency on another package and doesn't complain, right?